tests/unit/test_correlation.py

```
@@ -480,3 +480,97 @@ def test_all_functions_exported(self) -> None:
+        from unittest.mock import patch
+
+        from fragrance_rater.middleware.correlation import (
+            _correlation_id_ctx,
+            _request_id_ctx,
+            _trace_id_ctx,
+            configure_sentry_correlation,
+        )
+
+        captured = {}
+
+        def capture_processor(fn):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `capture_processor`

Add return type annotation: `None`

(ANN202)